Makefile

```diff
@@ -35,6 +35,9 @@ test: clean fmt REBUILD
 test-long: clean fmt REBUILD
 	go test -v -race -tags=test -timeout=180s ./...
 
+test-tpool: clean fmt REBUILD
+	go test -v -race -tags=test -timeout=8s github.com/NebulousLabs/Sia/modules/transactionpool
+
```

Comment:
do we need a separate target for this? `go test -v -race -tags=test ./transactionpool` seems succinct enough to just keep in history.